superbench/runner/runner.py

```diff
@@ -173,6 +178,72 @@ def fetch_results(self):    # pragma: no cover
             )
         )
 
+    def __create_results_summary(self):    # pragma: no cover
+        """Create the result summary files."""
+        all_results = list()
+        for node_path in (self._output_path / 'nodes').glob('*'):
+            if not node_path.is_dir():
+                continue
+            results_summary = dict()
+            reduce_ops = dict()
+            file_list = natsorted([str(f) for f in node_path.glob('**/results.json')])
+            file_list = [Path(f) for f in file_list]
+            for results_file in file_list:
+                with results_file.open() as f:
+                    results = json.load(f)
+                    for result in results:
+                        if result['name'] not in results_summary:
```

Comment:
Do we need to check `result['name']` is NULL or not? Because we will merge all `NULL` together.

Reply:
result['name'] is the name of benchmark, it is set when register the benchmark.